File: pid-backend/src/main/java/cz/fit/cvut/pidbackend/Model/Route.java
package cz.fit.cvut.pidbackend.Model;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "routes")
@Data
@NoArgsConstructor
@AllArgsConstructor(access = AccessLevel.PUBLIC)
public class Route {

    @Id
    @Column(name = "uid")
    private String id;

    @Column(name = "long_name")
    private String longName;
    @Column(name = "short_name")
    private String shortName;
//    @Column(name = "\"desc\"")
    @Column(name = "desc")
    private String desc;
    @Column(name = "agency")
    private String agency;
    @Column(name = "color")
    private String color;
    @Column(name = "text_color")
    private String textColor;
    @Column(name = "type")
    private String type;
    @Column(name = "url")
    private String url;
    @Column(name = "is_night")
    private boolean isNight;

    @Column(name = "shape_id")
    private String shapeId;
}

File: android-app/app/src/main/java/cvut/fit/pidmobapp/model/Route.kt
package cvut.fit.pidmobapp.model

import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.gson.annotations.SerializedName

@Entity
data class Route (
        @PrimaryKey
        @SerializedName("id")
        val uid: String,
        @SerializedName("longName")
        val longName: String,
        @SerializedName("shortName")
        val shortName: String,
        @SerializedName("desc")
        val desc: String? = "",
        @SerializedName("agency")
        val agency: String,
        @SerializedName("color")
        val color: String,
        @SerializedName("textColor")
        val textColor: String,
        @SerializedName("type")
        val type: String,
        @SerializedName("url")
        val url: String,
        @SerializedName("shapeId")
        val shapeId: String,
        @SerializedName("night")
        val isNight: Boolean
        )